lib/elixir/lib/inspect/algebra.ex

```diff
@@ -354,28 +347,24 @@ defmodule Inspect.Algebra do
             try do
               Process.put(:inspect_trap, true)
 
-              res =
-                Inspect.Map.inspect(struct, %{
-                  opts
-                  | syntax_colors: [],
-                    inspect_fun: Inspect.Opts.default_inspect_fun()
-                })
-
-              res = IO.iodata_to_binary(format(res, :infinity))
-
-              message =
-                "got #{inspect(caught_exception.__struct__)} with message " <>
-                  "#{inspect(Exception.message(caught_exception))} while inspecting #{res}"
-
-              exception = Inspect.Error.exception(message: message)
+              error_exception =
+                Inspect.Error.exception(
+                  exception: caught_exception,
+                  inspect_opts: opts,
```

Comment:
This option is not used.

Reply:
True. This is a left-over from previous experiments. Should we raise here?  we crash if the option is not valid, but these ones will go unnoticed.

Reply:
It is fine to not raise, it is all for internal use anyway. I would keep the code tidy rather than defensive. :)